src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnerServiceValidateTest.java

```diff
@@ -137,6 +137,31 @@ void shouldReturnErrorsWhenLimitedPartnerPersonDataIsInvalidAndCustomChecksFail(
                         tuple("At least one contribution type must be selected", LimitedPartnerDataDto.CONTRIBUTION_SUB_TYPES_FIELD));
     }
 
+    @ParameterizedTest
```

Comment:
Not much point this being a parameterised test as it only operates with one value.

Reply:
Changed this already